### PR TITLE
Add `gRPC` blockquote

### PR DESCRIPTION
**Pull request by ppcano**

### CODE REVIEW

File: src/components/shared/grpc-blockquote/grpc-blockquote.view.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Blockquote from 'components/shared/blockquote';
+import React from 'react';
+
+const GrpcBlockquote = () => (
```
Comment (olegbespalov) [inline src/components/shared/grpc-blockquote/grpc-blockquote.view.js:4]:
```suggestion
const GrpcExperimentalBlockquote = () => (
```

File: src/components/shared/grpc-blockquote/index.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import GrpcBlockquote from './grpc-blockquote.view';
+
+export default GrpcBlockquote;
```
Comment (olegbespalov) [inline src/components/shared/grpc-blockquote/index.js:3]:
```suggestion
import GrpcExperimentalBlockquote from './grpc-blockquote.view';

export default GrpcExperimentalBlockquote;
```

File: src/components/templates/doc-page/doc-page-content/doc-page-content.view.js [hunk 3]
```diff
@@ -12,6 +12,7 @@ import Collapsible from 'components/shared/collapsible';
 import CryptoBlockquote from 'components/shared/crypto-blockquote';
 import DescriptionList from 'components/shared/description-list';
 import ExperimentalBlockquote from 'components/shared/experimental-blockquote';
+import GrpcBlockquote from 'components/shared/grpc-blockquote';
```
Comment (olegbespalov) [inline src/components/templates/doc-page/doc-page-content/doc-page-content.view.js:15]:
```suggestion
import GrpcExperimentalBlockquote from 'components/shared/grpc-blockquote';
```

File: src/components/templates/doc-page/doc-page-content/doc-page-content.view.js [hunk 4]
```diff
@@ -45,6 +46,7 @@ const componentsForNativeReplacement = {
   BlockingAwsBlockquote,
   CryptoBlockquote,
   WsBlockquote,
+  GrpcBlockquote,
```
Comment (olegbespalov) [inline src/components/templates/doc-page/doc-page-content/doc-page-content.view.js:49]:
```suggestion
  GrpcExperimentalBlockquote,
```

File: src/data/markdown/docs/02 javascript api/11 k6-net-grpc.md [hunk 5]
```diff
@@ -3,16 +3,10 @@ title: "k6/net/grpc"
 excerpt: "k6 gRPC API"
 ---
 
-The `k6/net/grpc` module, added in k6 v0.29.0, provides a [gRPC](https://grpc.io/) client for Remote Procedure Calls (RPC) over HTTP/2.
+<GrpcBlockquote />
```
Comment (olegbespalov) [inline src/data/markdown/docs/02 javascript api/11 k6-net-grpc.md:6]:
```suggestion
<GrpcExperimentalBlockquote />
```

File: src/data/markdown/docs/02 javascript api/11 k6-net-grpc/10-Client.md [hunk 6]
```diff
@@ -3,10 +3,9 @@ title: Client
 excerpt: 'Client is a gRPC client that can interact with a gRPC server.'
 ---
 
-`Client` is a gRPC client that can interact with a gRPC server.
-
->  ⚠️ **Note**: Only unary RPCs are currently supported, i.e. there is no support for client, server or bidirectional streaming.
+<GrpcBlockquote />
```
Comment (olegbespalov) [inline src/data/markdown/docs/02 javascript api/11 k6-net-grpc/10-Client.md:6]:
```suggestion
<GrpcExperimentalBlockquote />
```

**Review by olegbespalov — COMMENTED** [review]
How about explicitly naming `GrpcExperimentalBlockquote` to keep the context of what this blockquote is about? WDYT?

**Comment by ppcano** [comment]
@olegbespalov, 

There are other similar cases: `cryptoBlockquote` and `wsBlockquote`. For consistency, I'd have to change to `cryptoExperimentalBlockquote` and `wsExperimentalBlockquote`. Not a big issue. 

But not sure because `grpcExperimentalBlockquote` could refer that the quote is for the `experimental/grpc` module, which is not the case. 

atm, I'll leave it as it is. But let me know if you have a strong opinion about it.

**Pull request merged**